### ISSUE TITLE
notify on items with buyout price lower than the vendor sell price

### ISSUE THREAD

**Issue by devlyn27** [Auctionator/Auctionator#919]
**Is your feature request related to a problem? Please describe.**
nope, no problems here!

**Describe the solution you'd like**
maybe an auto buy you can turn on in the settings or a list of said items to click through and buy

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by MyDevil94** [comment]
Yeah, that would be awsome.

**Comment by plusmouse** [comment]
A limited version is available in this custom Collectionator build. Go to the "Collecting" tab, click "Scan", and when the scan finishes click "Vendor".
[Collectionator-vendor-price-search.zip](https://github.com/Auctionator/Auctionator/files/5755027/Collectionator-vendor-price-search.zip)

**Comment by MyDevil94** [comment]
I've tested it. Works good. 
Maybe you can ignore out "only bid" auctions.
The tool display it with 0g/0s/0c

**Comment by sc1on1x** [comment]
I searched forever for something like this. So glad I stumbled across this file. (had to create a Discord account and a GitHub account in the process, yeah I'm an old dude) 

IDK if I did something wrong but I loaded this up and it was showing out-of-date. I deleted it, loaded the current Collectionator off of Curse. Then I loaded this file back and did "replace" and it fired right up. It seems a little glitchy (or maybe other people are buying these items up super quick) but I made 10g off of my first scan! So that's pretty awesome
